experiments/modern/src/maki-interpreter/virtualMachine.js

```diff
@@ -10,12 +10,11 @@ function coerceTypes (var1, var2, val1, val2) {
   return val1;
 }
 
-async function interpret(start, program, { logger = null }) {
+async function interpret(start, program, stack, { logger = null }) {
```

Comment:
Maybe we just give stack a default value of `[]`?

```
async function interpret(start, program, stack = [], { logger = null }) {
```

Reply:
i was going to, but then the places we use it pass in the 4th param anyway...guess thats not necessarily a reason to not include it though.